docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -439,29 +477,24 @@ If any of the assets in the module already has a `group_name` explicitly set on
 
 ### Viewing asset groups in Dagit
 
-To view your asset groups in Dagit, open the left navigation by clicking the **menu icon in the top left corner**. As asset groups are grouped in repositories, you may need to open a repository to view its asset groups:
+To view your asset groups in Dagit, open the left navigation by clicking the **menu icon in the top left corner**. As asset groups are grouped in code locations, you may need to open a code location to view its asset groups.
 
-<Image
-alt="Asset Groups in Dagit left navigation"
-src="/images/concepts/assets/asset-groups-in-sidenav.png"
-width={903}
-height={454}
-/>
+Click the asset group to open a dependency graph for all assets in the group. For example, in the following image, the dependency graph for the `activity_analytics` asset group is currently displayed:
 
-Click the asset group to open a dependency graph for all assets in the group:
+<!-- ![Dependency graph for an asset group](/images/concepts/assets/asset-group-dependency-graph.png) -->
```

Comment:
is this intentional? should we remove?

Reply:
It is intentional. It makes it easier to re-run the formatting script, in case the image's dimensions change. The script doesn't (at least from what I've seen) update images that are already using the `Image` component